### PR TITLE
5.1 docs updates

### PR DESCRIPTION
**Pull request by hflynn**
Adding a new limitation and some fixing and tidying ready for the release.

### CODE REVIEW

**Comment by manics** [comment]
What's not working?

**Comment by hflynn** [comment]
See new commit, this relates to https://trello.com/c/4Dgz8Hs1/88-weekly-testing (point 2 under 'Other' bottom checklist and Jason's comment)

File: omero/sysadmins/limitations.txt [hunk 1]
```diff
@@ -143,6 +143,15 @@ clients will use the extrema of the pixel type range by default. Users can
 adjust the minima/maxima via the rendering settings. Recalculating minima and
 maxima pixel values after import is currently not supported.
 
+Flex data with Acapella results in OMERO.tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are currently issues with the configuration required to access the raw
+data in combination with the analysis results via
+:doc:`OMERO.tables </developers/analysis>`. You may not be able to view the
+ROIs overlayed on the associated image by selecting the results they relate to
```
Comment (joshmoore) [inline omero/sysadmins/limitations.txt]: I'm not sure if there are any issues with the ROIs particularly. It's untested, but I don't think the code changed.

Comment (joshmoore) [inline omero/sysadmins/limitations.txt]: @pwalczysko : is this second warning from you?

File: omero/sysadmins/limitations.txt [hunk 2]
```diff
@@ -143,6 +143,15 @@ clients will use the extrema of the pixel type range by default. Users can
 adjust the minima/maxima via the rendering settings. Recalculating minima and
 maxima pixel values after import is currently not supported.
 
+Flex data with Acapella results in OMERO.tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are currently issues with the configuration required to access the raw
```
Comment (joshmoore) [inline omero/sysadmins/limitations.txt]: Perhaps something along the lines of "Anyone using the advanced configuration setting `FlexReaderServerMaps` for importing Flex data split between multiple directories should not upgrade to 5.1.0"?

Comment (hflynn) [inline omero/sysadmins/limitations.txt]: You happy with that @pwalczysko ? Sounds better to me!

File: omero/sysadmins/limitations.txt (hunk 1 above)
Comment (pwalczysko) [inline omero/sysadmins/limitations.txt]: @joshmoore No, not explicitly, I think we should go for what you suggest and ignore the ROIs. I was just trying to explain the feature but I did not know what is broken and what is not. Yes, agree, this is too much divined here. Probably leave out.

File: omero/sysadmins/limitations.txt (hunk 2 above)
Comment (pwalczysko) [inline omero/sysadmins/limitations.txt]: @hflynn : Definitely, go for it.

**Comment by hflynn** [comment]
@joshmoore @pwalczysko fix pushed - happy with this text now?

**Comment by joshmoore** [comment]
No objections.

**Comment by hflynn** [comment]
Commits 3 and 4 still need signing off on.

**Comment by joshmoore** [comment]
Still unbuilt, right?

**Comment by hflynn** [comment]
yeah, stupid Travis. I can rerun the merge build now if you like, it'll take about 25 mins.

**Comment by hflynn** [comment]
Building now

**Comment by joshmoore** [comment]
:thumbsup:

**Comment by hflynn** [comment]
--no-rebase

**Pull request merged**